_monitoring-plugins/alerting/security.md

```diff
@@ -82,3 +82,42 @@ If you only want users to be able to see and modify their own monitors and desti
 ```
 
 Then, use this new role for all alerting users. -->
+
+### Specify RBAC backend roles
+
+You can specify RBAC backend roles when you create or update a monitor with the Alerting API.
```

Comment:
might be worthwhile spelling out Rule Based Access Control here. I don't think it appears before this on the page.

Reply:
thanks. adding "role-based access control" with acronym in parentheses